<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>633</width>
    <height>576</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout" stretch="4,0,6">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout" stretch="5,5">
     <item>
      <widget class="QLabel" name="lb_image">
       <property name="minimumSize">
        <size>
         <width>300</width>
         <height>250</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>300</width>
         <height>250</height>
        </size>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="pixmap">
        <pixmap>../images/495646270_18141246325387864_8959956723308127787_n__1__13e2639671994fdea19966dee242d1da_grande.webp</pixmap>
       </property>
       <property name="scaledContents">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QLabel" name="lb_name">
         <property name="font">
          <font>
           <pointsize>12</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>Citlali Obsidian Opalstar Ver. - Genshin Impact | Myethos Figure</string>
         </property>
         <property name="wordWrap">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="lb_brand">
         <property name="text">
          <string>Thương hiệu: Myethos
</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="lb_price">
         <property name="font">
          <font>
           <pointsize>14</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">color: #ff0000;</string>
         </property>
         <property name="text">
          <string>1,650,000₫</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btnBuy">
         <property name="styleSheet">
          <string notr="true">height: 50px;
color:  #fff;
background: #ff0000;
border:none;
border-radius: 5px;</string>
         </property>
         <property name="text">
          <string>THÊM VÀO GIỎ HÀNG</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QLabel" name="label">
     <property name="font">
      <font>
       <pointsize>14</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>MÔ TẢ SẢN PHẨM</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QTextBrowser" name="textBrowser">
     <property name="styleSheet">
      <string notr="true">border:none;</string>
     </property>
     <property name="html">
      <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'MS Shell Dlg 2'; font-size:8pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:15px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-family:'Mulish,sans-serif'; font-weight:600; color:#000000; text-transform:uppercase;&quot;&gt;Mô tả sản phẩm&lt;/span&gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:13px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-family:'Mulish,sans-serif'; font-weight:496; color:#000000;&quot;&gt;Mô hình figure chính hãng &lt;/span&gt;&lt;span style=&quot; font-family:'Mulish,sans-serif'; font-weight:696; color:#000000;&quot;&gt;Citlali&lt;/span&gt;&lt;span style=&quot; font-family:'Mulish,sans-serif'; font-weight:496; color:#000000;&quot;&gt; Obsidian Opalstar Ver. Genshin Impact | Myethos&lt;/span&gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:15px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px; line-height:21px;&quot;&gt;&lt;span style=&quot; font-family:'Mulish,sans-serif'; font-weight:696; color:#000000;&quot;&gt;Citlali&lt;/span&gt;&lt;span style=&quot; font-family:'Mulish,sans-serif'; color:#000000;&quot;&gt;, một nhân vật trong &lt;/span&gt;&lt;span style=&quot; font-family:'Mulish,sans-serif'; font-weight:696; color:#000000;&quot;&gt;Genshin Impact&lt;/span&gt;&lt;span style=&quot; font-family:'Mulish,sans-serif'; color:#000000;&quot;&gt;, được Myethos sản xuất với phiên bản Obsidian Opalstar. Mô hình này được làm từ chất liệu ABS, Microfiber và PVC, với kích thước 1/8, cao 102mm (3.98 inches), và có tỉ lệ 1:1=0.82m. Đây là một sản phẩm tuyệt vời cho những fan của &lt;/span&gt;&lt;span style=&quot; font-family:'Mulish,sans-serif'; font-weight:696; color:#000000;&quot;&gt;Genshin Impact&lt;/span&gt;&lt;span style=&quot; font-family:'Mulish,sans-serif'; color:#000000;&quot;&gt;, đặc biệt là những ai yêu thích sự kết hợp giữa thiết kế mỹ thuật và chi tiết cao. Mô hình này sẽ trở thành một phần không thể thiếu trong bộ sưu tập của những người đam mê anime và game, mang lại không gian trưng bày đầy màu sắc và độc đáo. Được thiết kế tỉ mỉ đến từng chi tiết, từ bộ trang phục đến tạo hình nhân vật, sản phẩm này không chỉ là một vật phẩm trang trí mà còn là một tác phẩm nghệ thuật mà bất kỳ fan hâm mộ nào cũng muốn sở hữu.&lt;/span&gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:15px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px; line-height:21px;&quot;&gt;&lt;span style=&quot; font-family:'Mulish,sans-serif'; color:#000000;&quot;&gt;Để biết thêm thông tin và hỗ trợ mua hàng, bạn có thể tham khảo các liên kết sau:&lt;/span&gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:15px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px; line-height:21px;&quot;&gt;&lt;a href=&quot;https://jhfigure.com/pages/ship-quoc-te&quot;&gt;&lt;span style=&quot; font-family:'Mulish,sans-serif'; text-decoration: underline; color:#158ed4;&quot;&gt;Hỗ trợ ship quốc tế&lt;/span&gt;&lt;/a&gt;&lt;a href=&quot;https://jhfigure.com/pages/chinh-sach-doi-tra&quot;&gt;&lt;span style=&quot; font-family:'Mulish,sans-serif'; text-decoration: underline; color:#158ed4;&quot;&gt;Chính sách đổi trả&lt;/span&gt;&lt;/a&gt;&lt;a href=&quot;https://jhfigure.com/pages/cac-cau-hoi-thuong-gap-faq&quot;&gt;&lt;span style=&quot; font-family:'Mulish,sans-serif'; text-decoration: underline; color:#158ed4;&quot;&gt;Câu hỏi thường gặp&lt;/span&gt;&lt;/a&gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:15px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px; line-height:21px;&quot;&gt;&lt;span style=&quot; font-family:'Mulish,sans-serif'; font-weight:696; color:#000000;&quot;&gt;Citlali&lt;/span&gt;&lt;span style=&quot; font-family:'Mulish,sans-serif'; color:#000000;&quot;&gt; from &lt;/span&gt;&lt;span style=&quot; font-family:'Mulish,sans-serif'; font-weight:696; color:#000000;&quot;&gt;Genshin Impact&lt;/span&gt;&lt;span style=&quot; font-family:'Mulish,sans-serif'; color:#000000;&quot;&gt; is a beautifully crafted figure from Myethos in the Obsidian Opalstar Ver. Made from high-quality ABS, Microfiber, and PVC, this 1/8 scale figure stands at 102mm (3.98 inches). Perfect for collectors and fans of Genshin Impact, the figure captures every intricate detail of Citlali’s outfit and design. A must-have for anyone looking to expand their collection, this figure is not just a decoration but a true work of art that showcases the dedication to detail by Myethos. With its compact size, it fits perfectly into any collection, making it an essential piece for fans of the game.&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
